tests/integration/test_worker.py

```diff
@@ -215,3 +215,13 @@ def test_sigusr1(self):
             worker.expect('Consumer started')
             worker.send_signal(signal.SIGUSR1)
             worker.expect('traceback.format_stack')
+
+    def test_batch(self):
+        """Batch tasks are run"""
+        li = [tasks.echo.subtask(args=("foo %i" % i, )) for i in range(2)]
```

Comment:
Actually, I have looked into Celery sources and realized that they have implemented it the same way. In fact, they have two implementations:

* `.signature(args, kwargs)` (`.subtask` is an alias to `.signature`)
* `.s(*args, **kwargs)` (it, essentially, calls `.signature(args=args, kwargs=kwargs)`) + `.si(*args, **kwrags)` (they have a special case of "immutable" tasks)